organisation/collaboration.qmd

```diff
@@ -0,0 +1,87 @@
+---
+title: "Collaboration"
+---
+
+Collaboration is the basis for all human work and it is built from
+culture. Culture in general is something that is organic and develops on
+its own. It can be difficult to document or make explicit. Depending on
+the specific social dynamics of the team, of internal collaborators, and
+of external contributors, culture can change quite a fair amount. Many
+of these dynamics might happen through channels like in-person
+activities, issue discussions, or on online communication platforms.
+However, a healthy culture requires some self-awareness,
+self-reflection, and ultimately some self-regulation to remain healthy.
+Through the below principles, we hope to nurture a healthy culture of
+collaboration.
+
+## Recognize and account for the human aspect of work
+
+Humans are biological beings with limitations to their capacities with
+physical, emotional, and mental needs. Part of designing a
+healthy work culture is by recognizing and accounting for these limits
+and needs. We aim to design a work culture where we find and work on
+things that we enjoy doing, where we actively assess cognitive load and
+capacity when planning and designing products, and where we incorporate
+formal and informal social activities and support structures to help
+build relationships and trust.
+
+## Expect and anticipate openness and transparency
+
+A core value of the Seedcase Project is openness and transparency. This
+includes nearly all aspects of our work, including cross-collaborator
+communication. We aim to recognize, and anticipate, that we are working
+and communicating openly and that *anyone* could potentially see and
+read what we write. We also recognize that some things are private and
+should be kept private, and we aim to be clear about what those things
+are.
```

Comment:
```suggestion
should be kept private, which will be decided on a situational basis.
```

Maybe just make it easier and remove the "aim" part. Thinking about it, it makes more sense to describe (in other docs) what to put publicly and assume everything else is private.